### ISSUE TITLE
Add on-screen show/hide toggle

### ISSUE THREAD

**Issue by zreese** [Zero-to-Heroes/firestone#44]
The majority of the app is mouse-driven, so I'm curious why showing and hiding seems to only be possible using a key-binding. For accessibility purposes, it seems like a good idea to have some sort of floating button to toggle the visibility state. 

Do you accept pull requests? I'd love to contribute to this project.

**Comment by sebastientromp** [comment]
Hey, thanks for the suggestion!

When you are mentioning showing / hiding, what exactly are you talking about? The app's main window?

**Comment by zreese** [comment]
Yes, the thing that is toggled with Alt+C. I'm someone who uses a mouse
alone so I have to open up a virtual keyboard every time I want to toggle
it on/off. It would be a huge help for people like me if there was a
hotspot or button that could toggle the main window without needing a
keyboard.

**Comment by sebastientromp** [comment]
Ah, got it. I haven't implemented one because using the Firestone icon (from the desktop, or quick launch bar) should do what you want (bring up the main window).   
And hiding / closing the window can be done with the buttons at the top-right.

Not exactly the same as having a single widget in-game, but I feared that a permanent widget icon might be a bit annoying for most users.

**Comment by zreese** [comment]
Can you expand on that? I don't see a Firestone icon when I'm running
Hearthstone in full screen. If there's a way to show hide the main window
just using the mouse and I've been missing it this whole time, that'd
be great!

**Comment by sebastientromp** [comment]
I just use Windows key + type "Firestone", then I can use the icon to launch the app. But you're right, if you're full screen you also need the keyboard, so that defeats the whole purpose.

I hadn't thought of that case, I'll see if I can make something :)

**Comment by zreese** [comment]
Awesome!! Thanks!

________________________________

**Comment by sebastientromp** [comment]
Quick update: it's still on my todo list :)
